### ISSUE TITLE
[WIP] Add alt text to endnote arrow

### ISSUE THREAD

**Issue by pcraig3** [HHS/simpler-grants-pdf-builder#78]
@admoorgit to follow up.

**Comment by pcraig3** [comment]
@admoorgit I think we can close this?

**Comment by admoorgit** [comment]
@pcraig3 yeah — no issues were flagged by Kate so I assume what is there is good.

**Comment by pcraig3** [comment]
Closing this issue since it was more like a placeholder to check it out, and we checked it out and no issues were flagged.
